ida-ws/src/test/java/upb/ida/test/DataRepositoryTest.java

```diff
@@ -0,0 +1,56 @@
+package upb.ida.test;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+import upb.ida.Application;
+import upb.ida.dao.DataRepository;
+import java.util.Set;
+import java.util.TreeSet;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest
+@ContextConfiguration(classes = {Application.class})
+public class DataRepositoryTest {
+
+	private DataRepository dataRepository;
+
+	public DataRepositoryTest() {
+		dataRepository = new DataRepository(true);
+	}
+
+	@Test
+	public void getColumnListTestPos() {
+		Set<String> expected = new TreeSet<>();
+		expected.add("sameAs");
+		expected.add("capitalOf");
+		expected.add("hasCode");
+		expected.add("featureCode");
+		expected.add("featureClass");
+		expected.add("name");
```

Comment:
This piece of code looks repeated, maybe try for a better implementation?

Reply:
Using a class member instead of a local member now.